src/patternfly/components/FormControl/form-control.scss

```diff
@@ -65,26 +73,38 @@ $pf-c-form-control--m-search--Coordinates: "M505 442.7L405.3 343c-4.5-4.5-10.6-7
   --pf-c-form-control--success--PaddingBottom: calc(var(--pf-global--spacer--form-element) - var(--pf-c-form-control--success--BorderBottomWidth));
   --pf-c-form-control--success--BorderBottomColor: var(--pf-global--success-color--100);
   --pf-c-form-control--success--PaddingRight: var(--pf-global--spacer--xl);
-  --pf-c-form-control--success--BackgroundPosition: calc(100% - var(--pf-c-form-control--PaddingLeft)) var(--pf-c-form-control--PaddingLeft);
-  --pf-c-form-control--success--BackgroundSize: var(--pf-c-form-control--FontSize) var(--pf-c-form-control--FontSize);
+  --pf-c-form-control--success--BackgroundPositionX: calc(100% - var(--pf-c-form-control--PaddingLeft));
+  --pf-c-form-control--success--BackgroundPositionY: center;
+  --pf-c-form-control--success--BackgroundPosition: var(--pf-c-form-control--success--BackgroundPositionX) var(--pf-c-form-control--success--BackgroundPositionY);
+  --pf-c-form-control--success--BackgroundSizeX: var(--pf-c-form-control--FontSize);
+  --pf-c-form-control--success--BackgroundSizeY: var(--pf-c-form-control--FontSize);
```

Comment:
should this be: `--pf-c-form-control--success--BackgroundPositionX: var(--pf-c-form-control--success--BackgroundPositionY)` ?

Reply:
It could be... though if we do that, then `--pf-c-form-control--success--BackgroundSizeY` is defined as `--pf-c-form-control--success--BackgroundSizeX`. If we want `--pf-c-form-control--success--BackgroundSizeY` to be some unique value later on, changing it means it's no longer defined by `--pf-c-form-control--success--BackgroundSizeX` which could be considered a breaking change.

Reply:
got it! we can keep it like that then